cpp/src/parquet/column_reader_benchmark.cc

```diff
@@ -219,6 +251,20 @@ BENCHMARK(RecordReaderReadRecords)
     ->Args({2, 1000, true})
     ->Args({2, 1000, false});
 
+BENCHMARK(RecordReaderReadAndSkipRecords)
+    ->ArgNames({"Repetition", "BatchSize", "LevelsPerPage"})
+    ->Args({2, 1000, 80000})
+    ->Args({0, 1000, 80000})
+    ->Args({1, 1, 80000})
```

Comment:
@jp0317 Is there a reason for using such small batch sizes in the benchmarks?

Reply:
just wanted to simulate some extreme cases. I guess as long as the batch size is less than 1k it would show a noticeable perf difference